docs/tensorrt_plugin.md

```diff
@@ -0,0 +1,150 @@
+# TensorRT Plugins for custom operators in MMCV (Experimental)
+
+## Introduction of TensorRT
+
+**NVIDIA TensorRT** is a SDK for high-performance deep learning inference, which includes a deep learning inference optimizer and runtime that delivers low latency and high-throughput for deep learning inference applications. Check its [developer's website](https://developer.nvidia.com/tensorrt) for more information.
```

Comment:
1. a SDK -> a software development kit (SDK)
2. for high-performance inference of deep learning models.
3. It includes xxxxx.
4. Please check its